klog_test.go

```diff
@@ -1398,19 +1398,20 @@ func (l *testLogr) WithValues(...interface{}) logr.Logger {
 
 // existedFlag contains all existed flag, without KlogPrefix
 var existedFlag = map[string]struct{}{
-	"log_dir":           {},
-	"add_dir_header":    {},
-	"alsologtostderr":   {},
-	"log_backtrace_at":  {},
-	"log_file":          {},
-	"log_file_max_size": {},
-	"logtostderr":       {},
-	"one_output":        {},
-	"skip_headers":      {},
-	"skip_log_headers":  {},
-	"stderrthreshold":   {},
-	"v":                 {},
-	"vmodule":           {},
+	"add_dir_header":            {},
+	"alsologtostderr":           {},
+	"log_backtrace_at":          {},
+	"log_file":                  {},
+	"log_file_max_size":         {},
+	"klog_log_file_max_backups": {},
```

Comment:
New flag start with klog no need to be added in this map.

Reply:
correct, let's drop this @sayanibhattacharjee